Building fails

Unfortunately I know nothing about rust. Tried building both with --release and without.
```
cargo build  --verbose
       Fresh libc v0.2.17
       Fresh regex-syntax v0.3.9
       Fresh lazy_static v0.2.2
       Fresh getopts v0.2.14
       Fresh num_cpus v0.2.13
       Fresh ncurses v5.84.0
       Fresh memchr v0.1.11
       Fresh winapi v0.2.8
       Fresh winapi-build v0.1.1
       Fresh utf8-ranges v0.1.3
       Fresh aho-corasick v0.5.3
       Fresh kernel32-sys v0.2.2
       Fresh thread-id v2.0.0
       Fresh thread_local v0.2.7
       Fresh regex v0.1.80
   Compiling skim v0.1.2 (file:///home/quite/.skim)
     Running `rustc src/main.rs --crate-name sk --crate-type bin -g -C metadata=8c0cfc5409c9b61e --out-dir /home/quite/.skim/target/debug --emit=dep-info,link -L dependency=/home/quite/.skim/target/debug/deps --extern libc=/home/quite/.skim/target/debug/deps/liblibc-ad32fde1bd850538.rlib --extern getopts=/home/quite/.skim/target/debug/deps/libgetopts-33691dbdf8852281.rlib --extern regex=/home/quite/.skim/target/debug/deps/libregex-36c8e259ac5ba542.rlib --extern ncurses=/home/quite/.skim/target/debug/deps/libncurses-40460d940e0412dc.rlib --extern lazy_static=/home/quite/.skim/target/debug/deps/liblazy_static-7f1b96a3a3eb529d.rlib --extern num_cpus=/home/quite/.skim/target/debug/deps/libnum_cpus-9256729e2e3ab66d.rlib --extern skim=/home/quite/.skim/target/debug/deps/libskim.rlib`
error[E0554]: #[feature] may not be used on the stable release channel
 --> src/main.rs:1:1
  |
1 | #![feature(io)]
  | ^^^^^^^^^^^^^^^
```

Na... did I get a too fresh nightly on Arch Linux AUR? :)

```
error[E0308]: mismatched types
   --> src/model.rs:350:19
    |
350 |             getyx(stdscr, &mut y, &mut x);
    |                   ^^^^^^ expected *-ptr, found fn item
    |
    = note: expected type `*mut i8`
    = note:    found type `fn() -> *mut i8 {ncurses::stdscr}`
```

--- a/src/curses.rs
+++ b/src/curses.rs
@@ -133,7 +133,7 @@ impl Curses {
     pub fn get_maxyx(&self) -> (i32, i32) {
         let mut max_y = 0;
         let mut max_x = 0;
-        getmaxyx(stdscr, &mut max_y, &mut max_x);
+        getmaxyx(stdscr(), &mut max_y, &mut max_x);
         (max_y, max_x)
     }
 

--- a/src/model.rs
+++ b/src/model.rs
@@ -347,7 +347,7 @@ impl Model {
             // handle tabstop
             let mut y = 0;
             let mut x = 0;
-            getyx(stdscr, &mut y, &mut x);
+            getyx(stdscr(), &mut y, &mut x);
             let rest = (self.tabstop as i32) - (x-2)%(self.tabstop as i32);
             for _ in 0..rest {
                 self.curses.caddch(' ', color, is_bold);

